CoffeeMachineSimulator/CoffeeMachineSimulator.Services/Services/EspressoMachineService.cs

```diff
@@ -17,13 +17,19 @@ public EspressoMachineService(ICoffeeService coffeeService)
 
         public float GetSumOfAllCoffees()
         {
-            throw new System.NotImplementedException();
+            float sumOfAllPrices= 0f;
+            var coffees = coffeeService.GetCoffees();
+
+            foreach(CoffeeModel i in coffees)
```

Comment:
you can rename "i" with coffeeModel. in this case you are not using the index of the item in the list, you are making use of the object in your list.